public/sass/pages/_explore.scss

```diff
@@ -1,180 +1,312 @@
+}
 
-  .result-options {
-    margin: 2 * $panel-margin 0;
-  }
+.toolbar-content {
+  display: flex;
+  flex-flow: row wrap;
+  align-items: center;
+  justify-content: space-between;
+}
 
-  .time-series-disclaimer {
-    width: 300px;
-    margin: $panel-margin auto;
-    padding: 10px 0;
-    border-radius: $border-radius;
-    text-align: center;
-    background-color: $panel-bg;
-
-    .disclaimer-icon {
-      color: $yellow;
-      margin-right: $panel-margin/2;
-    }
+.toolbar-content-item {
+  padding: 10px 2px;
+}
 
-    .show-all-time-series {
-      cursor: pointer;
-      color: $external-link-color;
+.toolbar-content-item:first-child {
+  padding-left: $dashboard-padding;
+  margin-right: auto;
+}
+
+@media only screen and (max-width: 1545px) {
+  .toolbar.splitted {
+    .timepicker-rangestring {
+      display: none;
     }
   }
+}
 
-  .navbar .elapsed-time {
-    position: absolute;
-    left: 0;
-    right: 0;
-    top: 3.5rem;
-    text-align: center;
-    font-size: 0.8rem;
+@media only screen and (max-width: 1070px) {
+  .timepicker {
+    .timepicker-rangestring {
+      display: none;
+    }
   }
 
-  .graph-legend {
-    flex-wrap: wrap;
+  .toolbar-content {
+    justify-content: flex-start;
   }
 
-  .explore-panel__loader {
-    height: 2px;
-    position: relative;
-    overflow: hidden;
-    background: none;
-    margin: $panel-margin / 2;
-    transition: background-color 1s ease;
+  .toolbar.splitted {
+    .toolbar-content-item {
+      padding: 2px 0;
+      margin: 0;
+    }
   }
 
-  .explore-panel__loader--active {
-    background: $text-color-faint;
+  .toolbar-content-item {
+    padding: 2px 2px;
   }
+}
 
-  .explore-panel__loader--active:after {
-    content: ' ';
-    display: block;
-    width: 25%;
-    top: 0;
-    top: -50%;
-    height: 250%;
-    position: absolute;
-    animation: loader 2s cubic-bezier(0.17, 0.67, 0.83, 0.67);
-    animation-iteration-count: 100;
-    background: $blue;
+@media only screen and (max-width: 803px) {
+  .sidemenu-open {
+    .toolbar-header-title {
+      .navbar-page-btn {
+        padding-left: 0;
+        margin-left: 0;
+      }
+    }
   }
 
-  @keyframes loader {
-    from {
-      left: -25%;
-    }
-    to {
-      left: 100%;
+  .toolbar-header-title {
+    .navbar-page-btn {
+      padding-left: 0;
+      margin-left: $dashboard-padding;
     }
   }
 
-  .datasource-picker {
-    min-width: 200px;
+  .btn-title {
+    display: none;
   }
+}
 
-  .timepicker {
-    display: flex;
+@media only screen and (max-width: 702px) {
+  .toolbar-content-item:first-child {
+    padding-left: 2px;
+    margin-right: 0;
+  }
+}
 
-    &-rangestring {
-      margin-left: 0.5em;
+@media only screen and (max-width: 544px) {
+  .sidemenu-open {
+    .toolbar-header-title {
+      .navbar-page-btn {
+        padding-left: 0;
+        margin-left: $dashboard-padding;
+      }
     }
   }
 
-  .run-icon {
-    margin-left: 0.25em;
-    transform: rotate(90deg);
```

Comment:
the rotation got lost. Now the run arrow does no longer look like a Return key icon.

Reply:
My bad, good catch!